taca_ngi_pipeline/deliver/deliver.py

```diff
@@ -313,6 +314,53 @@ def create_report(self):
                 with_log_files=(logprefix is not None),
                 prefix="{}_aggregate".format(logprefix))
 
+    def copy_report(self):
+        """ Copies the aggregate report and version reports files to a specified outbox directory.
+            :returns: list of the paths to the files it has successfully copied (i.e. the targets)
+        """
+
+        def find_from_files_to_deliver(pattern):
+            """ Searches the nested list of `files_to_deliver` for files matching the provided pattern
+                :param pattern: the regex pattern to search for
+                :returns: single matching file
+                :raises: AssertionError if there is not strictly one match for the pattern
+            """
+
+            matches = []
+
+            for file_list in self.files_to_deliver:
+                for f in file_list:
+                    if re.match(pattern, f):
+                        matches.append(f)
+
+            if not matches or len(matches) != 1:
+                raise AssertionError("Found none of multiple matches for pattern: {}".format(pattern))
+            else:
+                return matches[0]
+
+        def create_target_path(target_file_name):
+            return self.expand_path(os.path.join(self.config["reports_outbox"], os.path.basename(target_file_name)))
```

Comment:
this will throw a `KeyError` if reports_outbox is not present in config, so I'd vote to either use `self.config.get(...)` or handle the exception upstream